fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalNestedLoopJoin.java

```diff
@@ -66,7 +69,7 @@ public String toString() {
         // TODO: Maybe we could pull up this to the abstract class in the future.
         StringBuilder sb = new StringBuilder();
         sb.append("PhysicalNestedLoopJoin ([").append(joinType).append("]");
-        condition.ifPresent(
+        otherJoinCondition.ifPresent(
```

Comment:
we also add equalCondition

Reply:
if we have equal condition, the logicalJoin will be translated to PhysicalHashJoin instead of PhysicalNestedLoopJoin.